src/view/login.js

```diff
@@ -0,0 +1,51 @@
+import Layout from "component/layout";
+
+var getUser = function(acct_id, users) {
+    var acctUser = users.filter(function(u) { return u.accountId() == acct_id; });
```

Comment:
I don't think it's so bad... classes and functions are :camel: and variables are :snake: